tensorboard/webapp/runs/views/runs_table/runs_table_component.scss

```diff
@@ -148,8 +144,8 @@ table {
 
 // Prevents the table column for checkbox and run_color from growing beyond
 // their sizes.
-.mat-column-checkbox,
-.mat-column-run_color {
```

Comment:
Ah, I was not aware that Angular Material automatically created classnames based on the provided column text and attached them.  I now see why we need the `tb-column-*` analogs.